### PR TITLE
docs: create entry for getstakeversions in rpcapi

### LINKED ISSUE #532: The new getstakeversions RPC needs to be added to docs/json_rpc_api.md
As the title indicates, the new `getstakeversions` RPC was added without also adding it to the documentation for the [JSON-RPC API](https://github.com/decred/dcrd/blob/master/docs/json_rpc_api.md).

This should have been caught during review.

### PR DESCRIPTION
**Pull request by dnldd**
Fixes #532

### CODE REVIEW

File: docs/json_rpc_api.md [hunk 1]
```diff
@@ -675,6 +676,16 @@ The following is an overview of the RPC methods which are implemented by dcrd, b
 |Returns|`[ (json array of strings)` <br/>&nbsp;&nbsp; `"blockhash", ... hash of the generated block` <br/>`]` |
 [Return to Overview](#MethodOverview)<br />
 
+<a name="getstakeversions"/>
+
+|   |   |
+|---|---|
+|Method|getstakeversions|
+|Parameters|1. hash (*chainhash.Hash, required) - The hash of the block being queried for stake versions |
```
Comment (davecgh) [inline docs/json_rpc_api.md]: This should b string.  You can't send a `*chainhash.Hash` over RPC.

Comment (davecgh) [inline docs/json_rpc_api.md]:
```
$ dcrctl help getstakeversions
getstakeversions "hash" count

Returns the stake versions statistics.

Arguments:
1. hash  (string, required)  The start block hash.
2. count (numeric, required) The number of blocks that will be returned.

Result:
{
 "stakeversions": [{ (array of object) Array of stake versions per block.
  "hash": "value",   (string)          Hash of the block.
  "height": n,       (numeric)         Height of the block.
  "blockversion": n, (numeric)         The block version
  "stakeversion": n, (numeric)         The stake version of the block
  "votes": [{        (array of object) The version and bits of each vote in the block
   "version": n,     (numeric)         The version of the vote.
   "bits": n,        (numeric)         The bits assigned by the vote.
  },...],
 },...],
}
```

Comment (davecgh) [inline docs/json_rpc_api.md]: As you can see you're missing the optional count and the returned result isn't specified.

Comment (dnldd) [inline docs/json_rpc_api.md]: updating the doc, will be converting the result section to the new format.

**Pull request merged**